Makefile

```diff
@@ -57,6 +57,14 @@ run-build run-rebuild: run-down
 # stop any containers, rebuild containers, and start it again
 run-build-start: run-rebuild run-start
 
+lint run-lint: 
+	PATH="${PWD}/venv/bin:${PATH}"
+	tox -e bandit
+	tox -e black
+	tox -e isort
+	tox -e flake8
+	tox -e mypy
```

Comment:
You can also run them all with one tox command (might be easier to see the failures that way)

`tox -e bandit,black,...`